jina/types/request/dump.py

```diff
@@ -0,0 +1,37 @@
+from . import Request
+
+
+class DumpRequest(Request):
+    """A request telling an Indexer to dump its data"""
+
+    @property
+    def path(self):
+        """
+
+
+        .. # noqa: DAR102
+
+
+        .. # noqa: DAR201
+        """
+        return self.body.path
+
+    @path.setter
+    def path(self, value):
+        self.body.path = value
+
+    @property
+    def shards(self):
+        """
+
+
+        .. # noqa: DAR102
+
+
+        .. # noqa: DAR201
+        """
+        return self.body.shards
+
+    @shards.setter
+    def shards(self, value):
+        self.body.shards = value
```

Comment:
The number of shards in dumping should be consistent with the shards in querying time, right?

Reply:
Yes